.circleci/config.yml

```diff
@@ -71,15 +71,22 @@ jobs:
   # Run parsing stats and publish them to the semgrep dashboard.
   autofix-printing-stats:
     docker:
-      - image: returntocorp/semgrep-dev:develop
+      - image: ocaml/opam:ubuntu
     steps:
       - checkout
       - run:
           name: autofix printing stats
           no_output_timeout: 60m
           command: |
-            cd autofix-printing-stats
-            ./run --upload
+            sudo apt-get update
+            sudo apt-get install -y python3-pip jq
+            pip3 install semgrep
+            # Put semgrep-core on the path. This is a little bit shady but
+            # apparently the `pip show` output is meant to be machine-parseable
+            # so this output should be stable:
+            # https://github.com/pypa/pip/issues/5261
+            PATH=$PATH:"$(pip3 show semgrep | grep '^Location' | sed -e 's/Location: //')/semgrep/bin"
+            opam exec -- ./autofix-printing-stats/run --upload
```

Comment:
I'd rather publish a docker image corresponding to the end of the first stage for our multistage build, like I suggested in Slack.

Reply:
I agree that that would be a better long-term fix. I'll add a TODO here for that, but I'd prefer not to get derailed with that task right now, especially since I am not very familiar with Docker.